pyomo/contrib/alternative_solutions/tests/test_lp_enum.py

```diff
@@ -0,0 +1,100 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2024
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+from pyomo.common.dependencies import numpy as numpy, numpy_available
+
+import pyomo.environ as pe
+from pyomo.common import unittest
+import pyomo.opt
+
+import pyomo.contrib.alternative_solutions.tests.test_cases as tc
+from pyomo.contrib.alternative_solutions import lp_enum
+
+#
+# Find available solvers. Just use GLPK if it's available.
+#
+solvers = list(
+    pyomo.opt.check_available_solvers("glpk", "gurobi")
+)  # , "appsi_gurobi"))
+pytestmark = unittest.pytest.mark.parametrize("mip_solver", solvers)
+
+timelimit = {"gurobi": "TimeLimit", "appsi_gurobi": "TimeLimit", "glpk": "tmlim"}
+
+
+@unittest.pytest.mark.default
+class TestLPEnum:
+
+    def Xtest_no_time(self, mip_solver):
```

Comment:
Same comment here--it might be good to skip it and leave a breadcrumb about fragility.

Reply:
Done